docs/modules/node.md

```diff
@@ -420,7 +421,8 @@ Redirects the Lua interpreter to a `stdout` pipe when a CB function is specified
 
 #### Example
 
-See the `telnet/telnet.lua` in `lua_examples` for a more comprehensive example of its use.
+See [the telnet lua module](../lua_modules/telnet.md) for a worked
```

Comment:
Unfortunately the link has to be lua-modules, not with underscore. Don't know how we got that in. Or we rename the docs/lua-modules folder.